pkg/apis/clickhouse.altinity.com/v1/type_config_chop.go

```diff
@@ -617,7 +618,11 @@ func (config *OperatorConfig) GetInformerNamespace() string {
 
 		// This contradicts current implementation of multiple namespaces in config's watchNamespaces field,
 		// but k8s has possibility to specify one/all namespaces only, no 'multiple namespaces' option
-		namespace = config.WatchNamespaces[0]
+		const labelFmt string = "[a-z]([-a-z0-9]*[a-z0-9])?"
+		var labelRegexp = regexp.MustCompile("^" + labelFmt + "$")
```

Comment:
maybe 
```
var labelRegexp = regexp.MustCompile("^[a-z]([-a-z0-9]*[a-z0-9])?$")
```
will better?

Reply:
hmm, ok and this just referring to the K8S format.